modules/commandModule/commandModule.py

```diff
@@ -404,6 +465,248 @@ def set_begin_takeoff(self, beginTakeoff: bool):
         self.__pigoData.update({"beginTakeoff" : beginTakeoff})
         self.__write_to_pigo_file()
 
+    def set_num_waypoints(self, numWaypoints: int):
+        """
+
+        Parameters
+        ----------
+        numWaypoints: int
+
+        """
+        if numWaypoints is None:
+            self.__logger.error("numWaypoints must be an int and not None.")
+            return None
+        if type(numWaypoints) is not int:
+            self.__logger.error("numWaypoints must be an int and not {}.".format(type(numWaypoints)))
+            return None
+        self.__pigoData.update({"numWaypoints": numWaypoints})
+        self.__write_to_pigo_file()
+
+    def set_waypoint_modify_flight_path_command(self, waypointModifyFlightPathCommand: int):
+        """
+
+        Parameters
+        ----------
+        waypointModifyFlightPathCommand: int
+
+        """
+        if waypointModifyFlightPathCommand is None:
+            self.__logger.error("waypointModifyFlightPathCommand must be an int and not None.")
+            return None
+        if type(waypointModifyFlightPathCommand) is not int:
+            self.__logger.error("waypointModifyFlightPathCommand must be an int and not {}.".format(type(waypointModifyFlightPathCommand)))
+            return None
+        self.__pigoData.update({"waypointModifyFlightPathCommand": waypointModifyFlightPathCommand})
+        self.__write_to_pigo_file()
```

Comment:
Something to think about for later, we keep rewriting the entire pigo file when really we should just be updating it.